ivanti_security_controls/plugin.spec.yaml

```diff
@@ -226,3 +454,110 @@ actions:
         description: Agent status with details
         type: agent_status
         required: true
+  start_patch_scan:
+    title: Start Patch Scan
```

Comment:
```suggestion
    title: Start a Patch Scan
```

Reply:
I think the former is fine here for a title